massa-protocol-worker/src/handlers/endorsement_handler/retrieval.rs

```diff
@@ -243,7 +243,7 @@ pub(crate) fn note_endorsements_from_peer(
     // From there we note new endorsements and propagate them
 
     // Filter out endorsements if they are too old (max age of the inclusion slot: `max_endorsements_propagation_time`)
-    let now = MassaTime::now()?;
```

Comment:
can the error variant corresponding to this "?" be removed as well ?

Reply:
No, used [here](https://github.com/massalabs/massa/blob/5f46612d74b4080ffeb83878b7e17f40b8c8d7fa/massa-protocol-worker/src/handlers/block_handler/retrieval.rs#L931) and [here](https://github.com/massalabs/massa/blob/b4e352543ddd75a82c9ca78fbc8ce548d951c04a/massa-protocol-worker/src/handlers/operation_handler/retrieval.rs#L256)